exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
```

Comment:
Hi Negin, Well done!
Your code is correct, also there is  another way you can try, it might be easier when using more than one variables.
let message= `Hello my name is ${myName} and my name is ${lengthName} character long`;

Reply:
Somayeh's suggestion is a good one to make your code more simple. The method is called string interpolation. https://www.freecodecamp.org/news/javascript-string-format-how-to-use-string-interpolation-in-js/

Be careful, when you use string interpolation in Javascript you have to wrap the string in backticks(`),  not single quotes (').